api/open-zoom-meeting.js

```diff
@@ -115,15 +115,35 @@ export default async ({ creatorSlackID, isHackNight } = {}) => {
     });
   }
 
-  const hostKey = Math.random().toString().substr(2, 6).padEnd(6, 0);
+  let hostKey = Math.random().toString().substr(2, 6).padEnd(6, 0);
 
   // sendHostKey({creatorSlackID, hostKey})
 
-  await zoom.patch({
+  // attempt to set the host key
+  const keyResponse = await zoom.patch({
     path: `users/${host.zoomID}`,
     body: { host_key: hostKey },
   });
 
+  // update the host record with the new key
+  if (keyResponse.http_code !== 400) {
```

Comment:
Shouldn't this be flipped, checking only for 2xx?  What happens if a 500 is returned?

Reply:
zoom will return a 204 (No Content) response if this is successful